apps/src/lab2/levelEditors/validations/EditValidation.tsx

```diff
@@ -94,6 +94,22 @@ const EditValidation: React.FunctionComponent<EditValidationProps> = ({
           }}
         />
       </div>
+      <div className={moduleStyles.row}>
+        <label htmlFor="callout" className={moduleStyles.label}>
+          Callout (optional):
+        </label>
+        <textarea
+          id="callout"
```

Comment:
nit: This id looks generic to me, afraid of possible collisions in the future. Maybe use something more specific like `levelEditors_editValidation_callout`

Reply:
Updated.  (Though not mixing camel case & snake case, which worries me. :)